diff --git a/.husky/pre-commit b/.husky/pre-commit
index 58e1758..69708f7 100644
--- a/.husky/pre-commit
+++ b/.husky/pre-commit
@@ -42,0 +43 @@ while IFS= read -r file; do
+    # Ensure the line numbers are captured correctly
@@ -66 +67,3 @@ if [ -f filtered-scanner-report.csv ]; then
-  if [ "$severity_fail_count" -gt 0 ]; then
+
+  # Fix to prevent the integer expression error
+  if [ -n "$severity_fail_count" ] && [ "$severity_fail_count" -gt 0 ]; then
@@ -74 +77 @@ else
-fi
+fi
\ No newline at end of file
diff --git a/force-app/main/default/classes/Account_Insert.cls b/force-app/main/default/classes/Account_Insert.cls
index cdd427c..a6046c2 100644
--- a/force-app/main/default/classes/Account_Insert.cls
+++ b/force-app/main/default/classes/Account_Insert.cls
@@ -21 +21 @@ public with sharing class AccountInsertClass {
-          System.debug('Hello Hi');
+          System.debug('Hello');
